code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -1,19 +1,41 @@
 /* Food */
+/datum/reagent/kois_clean
+	name = "Filtered K'ois"
+	id = "koispasteclean"
+	description = "A filtered, ketchup-like substance, filled with K'ois nutrients."
+	metabolism = REM * 4
+	var/nutriment_factor = 10
+	var/injectable = 0
+	color = "#ece9dd"
+	taste_description = "boiled cabbage"
+	unaffected_species = IS_MACHINE
+
+/datum/reagent/kois_clean/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
+	if(isvaurca(M))
+		M.heal_organ_damage(1.2 * removed, 1.2 * removed)
+		M.adjustToxLoss(-1.2 * removed)
+		M.nutrition += nutriment_factor * removed // For hunger and fatness
+		M.add_chemical_effect(CE_BLOODRESTORE, 6 * removed)
+	else
+		M.adjustToxLoss(1 * removed)
+		return
+	..()
+
 /datum/reagent/kois
-	name = "K'ois"
+	name = "Unfiltered K'ois"
 	id = "koispaste"
 	description = "A thick goopy substance, rich in K'ois nutrients."
 	metabolism = REM * 4
-	var/nutriment_factor = 10
+	var/nutriment_factor = 20
 	var/injectable = 0
 	color = "#dcd9cd"
 	taste_description = "boiled cabbage"
 	unaffected_species = IS_MACHINE
 
 /datum/reagent/kois/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
 	if(isvaurca(M))
-		M.heal_organ_damage(1.2 * removed, 1.2 * removed)
-		M.adjustToxLoss(-1.2 * removed)
+		M.heal_organ_damage(2.4 * removed, 2.4 * removed)
```

Comment:
Ah, the classic tactic of sneaking in ridiculous buffs along with a moderate nerf. Any particular reason we need to double K'ois' healing efficacy? If you want to make the two distinct, then nerf the clean K'ois'. 2.4*removed is simply ridiculous, especially when unfiltered K'ois is the _easy_ reagent to procure.

Reply:
most antitoxin medicine heals 4-6 units, but I'll revert it if it's an issue

Reply:
Most anti-toxin also doesn't simultaneously heal brute and burn damage.